### ISSUE TITLE
As an OFA admin, I can adjust user roles

### ISSUE THREAD

**Issue by RafterGit** [raft-tech/TANF-app#105]
#### Description:
This issue will deliver OFA admin adding, removing, and updating permissions of user roles 

AC:

**Comment by amilash** [comment]
@abottoms-coder  Is this duplicative work of the TDP permissions epic work?  #888
Or this related to a subsection of that work? or is this more related to Django admin stuff? 
Im not sure if we should close this one- or add these tickets to that other epic.

**Comment by amilash** [comment]
Closing this epic and open subtasks and they are based on API work and an old technical direction. We now have Django admin console and dont need this work. We can open new tickets as needed once we decide to move the django admin to the app UI. 

As per @jtwillis92 	
jwillis
9:50 AM
I believe that ticket was before we moved to using Django admin when this stuff was going to be built out with APIs and the UI - we can do all of that stuff in Django admin so I think that epic can be closed

	
angelamilash
9:51 AM
Thanks @jwillis . There are 4 tickets that are unfinished within this epic... are they actually completed? or is it work thats outstanding for Django? 

9:52 AM
I cant tell if those are dupes of work we will do with the new perms epic.

	
jwillis
9:54 AM
we don't need those - they are referring to API endpoints to do those operations

	
angelamilash
9:55 AM


Okay, so I will close as- already built w/in django
